docs/getting-started-guides/rkt/index.md

```diff
@@ -2,108 +2,98 @@
 ---
 
 This document describes how to run Kubernetes using [rkt](https://github.com/coreos/rkt) as a container runtime.
-We still have [a bunch of work](http://issue.k8s.io/8262) to do to make the experience with rkt wonderful, please stay tuned!
 
 ### **Prerequisite**
 
-- [systemd](http://www.freedesktop.org/wiki/Software/systemd/) should be installed on the machine and should be enabled. The minimum version required at this moment (2015/09/01) is 219
+- [systemd](http://www.freedesktop.org/wiki/Software/systemd/) should be installed on the machine and should be enabled.
+  The minimum version required for Kubernetes 1.3 is `219`.
   *(Note that systemd is not required by rkt itself, we are using it here to monitor and manage the pods launched by kubelet.)*
 
 - Install the latest rkt release according to the instructions [here](https://github.com/coreos/rkt).
-  The minimum version required for now is [v0.8.0](https://github.com/coreos/rkt/releases/tag/v0.8.0).
+  The minimum version required is [v1.9.0](https://github.com/coreos/rkt/releases/tag/v1.9.0).
 
-- Note that for rkt version later than v0.7.0, `metadata service` is not required for running pods in private networks. So now rkt pods will not register the metadata service be default.
-
-- Since release [v1.2.0-alpha.5](https://github.com/kubernetes/kubernetes/releases/tag/v1.2.0-alpha.5),
-the [rkt API service](https://github.com/coreos/rkt/blob/master/api/v1alpha/README.md)
-must be running on the node.
+- The [rkt API service](http://coreos.com/rkt/docs/latest/subcommands/api-service.html) must be running on the node.
 
 ### Network Setup
 
-rkt uses the [Container Network Interface (CNI)](https://github.com/appc/cni)
-to manage container networking. By default, all pods attempt to join a network
-called `rkt.kubernetes.io`, which is currently defined [in
-`rkt.go`](https://github.com/kubernetes/kubernetes/blob/v1.2.0-alpha.6/pkg/kubelet/rkt/rkt.go#L91).
-In order for pods to get correct IP addresses, the CNI config file must be
-edited to add this `rkt.kubernetes.io` network:
-
-#### Using flannel
+You can configure your network using the kubenet and CNI [network
+plugins](http://kubernetes.io/docs/admin/network-plugins/) by setting the
+`--network-plugin` flag, as described in each plugin's documentation.
 
-In addition to the basic prerequisites above, each node must be running
-a [flannel](https://github.com/coreos/flannel) daemon. This implies
-that a flannel-supporting etcd service must be available to the cluster
-as well, apart from the Kubernetes etcd, which will not yet be
-available at flannel configuration time. Once it's running, flannel can
-be set up with a CNI config like:
 
-```shell
-$ cat <<EOF >/etc/rkt/net.d/k8s_cluster.conf
+In addition, rkt supports using rkt's [Contained Networking](https://coreos.com/rkt/docs/latest/networking.html#contained-mode).
```

Comment:
We should add a description how to actually enable rkt contained networking in the kubelet.